<ui version="4.0" >
 <class>ConnectionOptionsWidgetClass</class>
 <widget class="QWidget" name="ConnectionOptionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>67</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbOptions" >
     <property name="title" >
      <string>Connection</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QWidget" native="1" name="wdtSelectConnection" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="lblConnection" >
             <property name="text" >
              <string>Connection:</string>
             </property>
             <property name="buddy" >
              <cstring>cmbConnections</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbConnections" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
